src/main/java/pl/tomaszdziurko/jvm_bloggers/mailing/BlogPostForMailItem.java

```diff
@@ -51,15 +52,14 @@ public Builder withIssueNumber(long issueNumber) {
         }
 
         public BlogPostForMailItem build() {
-            Preconditions.checkState(instance.title != null, "Tittle cannot be null");
-            Preconditions.checkState(instance.url != null, "Url cannot be null");
-            Preconditions.checkState(instance.authorLabel != null, "Author cannot be null");
-
+            Preconditions.checkState(StringUtils.isNotBlank(instance.title), "Tittle cannot be null");
+            Preconditions.checkState(StringUtils.isNotBlank(instance.url), "Url cannot be null");
+            Preconditions.checkState(StringUtils.isNotBlank(instance.authorLabel), "Author cannot be null");
```

Comment:
You have updated preconditions but messages are still outdated and misleading. It should be sth. like  `... cannot be null nor empty`.